theano/gpuarray/tests/test_pool.py

```diff
@@ -47,6 +47,19 @@ def test_pool_c_interface(self):
             f = theano.function([], ds_op(inp, [2, 2], pad=pad), mode=gpu_mode)
             f()
 
+    def test_pool_big_ws(self):
+        gpu_mode = copy.copy(mode_with_gpu).excluding("cudnn")
```

Comment:
why copy.copy()? Normlally just doing mode_with_gpu.excluding('cudnn') should do the work.